lib/plugins/aws/package/compile/events/api-gateway/lib/rest-api.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
         Properties: {
           Name: this.provider.naming.getApiGatewayName(),
           BinaryMediaTypes,
+          Description: this.provider.getApiGatewayDescription(),
```

Comment:
Let's only assign it if it's actually defined - it will allow us to avoid these validations where `Description` is undefined

Reply:
I think the function "getApiGatewayDescription() " checks whether description property exists.
https://github.com/serverless/serverless/blob/8bc54f968c4a4f208e8070f36dbe90d392a92c87/lib/plugins/aws/provider.js#L1756

If checking logic is added around the line you commented, it might be redundant check...?
(I'm sorry if I misunderstand.)

Reply:
Oh sorry. I understood what you mean.
You mean that we should avoid generating cloudformation file with "Description: undefined".

Reply:
I added the logic if it’s defined with following existing checking logic. (DisableExecuteApiEndpoint property)

I confirmed it can be deployed to aws either it’s defined or not.
Thank you for giving me precise comments!